conf/config.toml

```diff
@@ -66,9 +66,15 @@ region-schedule-limit = 2048
 replica-schedule-limit = 64
 merge-schedule-limit = 8
 hot-region-schedule-limit = 4
+
 ## There are some strategics supported: ["count", "size"], default: "count"
-# leader-schedule-strategy = "count" 
-## When the score difference between the leader or Region of the two stores is 
+# leader-schedule-strategy = "count"
+
+## When add new store, get the maximum capacity in the cluster, and then let max_score = max_capacity + flexible_score.
+## If capacity of the new store is lower than max_score, it will not be adjusted.
+## Otherwise, we will set the max_score of all the store to the new one.
+# flexible-score = 10485760
```

Comment:
Maybe we can remove it from this file. It's hard for users to understand what it means.

Reply:
Good